project/config/communityrelations/config/structure_sync.data.yml

```diff
@@ -493,3 +528,51 @@ menus:
     weight: -46
     langcode: en
     uuid: 1e27f577-473c-4aa8-9411-abd8fc2ae52c
+  -
```

Comment:
These menu entries in structure_sync won't do any harm, but they won't get imported either (only blocks and taxonomies are imported)